### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by Young-Gaius**
In this milestone, I did the following:

Deleted all the boilerplate from Create React App (text, images, styles).
Created a new directory called Components.
Inside components, created a new Calculator.js file.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <button type="button">3</button>
+      <button type="button" className="operator">+</button>
+    </div>
+    <div className="row">
+      <button type="button">0</button>
+      <button type="button">.</button>
+      <button type="button" className="equals">=</button>
+    </div>
+  </div>
+);
```
Comment (DammyShittu) [inline src/components/Calculator.js]: - [ ] The Calculator component should have at least one child component. Kindly add this component.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import './Calculator.css';
+
+const Calculator = () => (
+  <div className="calculator">
+    <div className="display">0</div>
+    <div className="row">
+      <button type="button" className="operator">AC</button>
+      <button type="button" className="operator">+/-</button>
+      <button type="button" className="operator">%</button>
+      <button type="button" className="operator">+</button>
```
Comment (DammyShittu) [inline src/components/Calculator.js]: - [ ] You have two plus signs. This should be &div; as contained in the design.

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './Calculator.css';
+
+const Calculator = () => (
```
Comment (DammyShittu) [inline src/components/Calculator.js:5]: - [ ] Your design should match the one provided. Kindly redesign the UI to match the design. It is important as front-end developers that we learn to stick to the provided designs.

  Your Implementation | Figma Design
  :---:|:---:
  image | image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Young-Gaius,

## Highlights
✔️ Deleting generated files

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Button.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Button() {
+  return (
+    <div className="num">0</div>
+  );
+}
+
+export default Button;
```
Comment (ghost) [inline src/components/Button.js]: - [ ] 👋 Hey there! 😄 Just a friendly heads-up – I noticed you crafted a cool subcomponent (button) for the calculator, but it's not being used in the main component. Let's get that subcomponent shining by importing and integrating it. This way, we'll boost efficiency and maintainability. Keep up the awesome work! 💪😄

**Review by ghost — CHANGES_REQUESTED** [review]
Hi @Young-Gaius , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ calculator is designed as expected 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by gemmen29 — APPROVED** [review]
Hi @Young-Gaius, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**